services/api/src/routes/users.js

```diff
@@ -65,6 +57,49 @@ router
       };
     }
   )
+  .post(
+    '/:userId/impersonate',
```

Comment:
Can we call this something other than "impersonate"? I may have been the one that suggested this naming but after thinking about it I think this word carries connotations of hacking and security holes that could easily make consumers uneasy.

I'm thinking something like "proxy", ie `proxyRoles` in roles.json and `POST /1/users/proxy` etc...

Reply:
I think proxy sounds too abstract wouldnt know what it means, but i get your point let me talk to dom and see if got a good name

Reply:
I think we should keep it simple: We are authenticating as a user. So we can call this `:userId/authenticate`

The audit trail can be called "Authenticated as user"

Agree that impersonate implies nefarious activity. Use of this feature can be compliant if configured correctly